docs/ossm/kiali-multi-cluster/README.md

```diff
@@ -0,0 +1,137 @@
+
+### Procedure
+
+1. Install Kiali on the `east` cluster.
+
+   Create a file named `kiali.yaml`.
+
+   ```yaml
+   apiVersion: kiali.io/v1alpha1
+   kind: Kiali
+   metadata:
+     name: kiali
+     namespace: istio-system
+   spec:
+     version: v1.89
+   ```
+
+   Apply the yaml file into the `east` cluster.
+
+   ```sh
+   oc --context east apply -f kiali.yaml
+   ```
+
+   Wait for Kiali to become ready.
+
+   ```sh
+   oc --context east wait --for=condition=Successful --timeout=60s kialis/kiali -n istio-system
+   ```
+
+1. Create an `OAuthClient` on the remote cluster so that Kiali can access the OpenShift API server on behalf of users.
+
+   Find your Kiali URL
+
+   ```sh
+   oc --context east get route kiali -n istio-system -o jsonpath='{.spec.host}'
+   ```
+
+   Create a file named `oauthclientwest.yaml`
+
+   ```yaml
+   apiVersion: oauth.openshift.io/v1
+   grantMethod: auto
+   kind: OAuthClient
+   metadata:
+     labels:
+       app: kiali
+       app.kubernetes.io/instance: kiali
+       app.kubernetes.io/name: kiali
+       app.kubernetes.io/part-of: kiali
+     name: kiali-istio-system
+   redirectURIs:
+     - https://<your-kiali-route>/api/auth/callback/west
+   ```
+
+   Create the `OAuthClient` in the west cluster.
+
+   ```sh
+   oc --context west apply -f oauthclientwest.yaml
+   ```
+
+1. Create a remote cluster secret.
+
+   In order to access a remote cluster, you must provide a kubeconfig to Kiali via a Kubernetes secret. You can use [this script](https://raw.githubusercontent.com/kiali/kiali/master/hack/istio/multicluster/kiali-prepare-remote-cluster.sh) to simplify this process for you. Running this script will:
+
+   - Create a Service Account for Kiali in the remote cluster.
+   - Create RBAC resources for this Service Account in the remote cluster.
+   - Create a kubeconfig file and save this as a secret in the namespace where Kiali is deployed on the `east` cluster.
+
+   1. Download the `kiali-prepare-remote-cluster` script.
+
+      ```sh
+      curl -L -o kiali-prepare-remote-cluster.sh https://raw.githubusercontent.com/kiali/kiali/master/hack/istio/multicluster/kiali-prepare-remote-cluster.sh
+      ```
+
+   2. Make the script executeable.
+
+      ```sh
+      chmod +x kiali-prepare-remote-cluster.sh
+      ```
+
+   3. Run the script passing your `east` and `west` cluster contexts.
+
+      ```sh
+      ./kiali-prepare-remote-cluster.sh --kiali-cluster-context east --remote-cluster-context west --view-only false --kiali-resource-name kiali --remote-cluster-namespace istio-system --remote-cluster-name west
```

Comment:
In my 2-cluster setup, I got this error message after running this (which makes sense):
```
ERROR: The remote cluster named [api-mazzwest-lab-upshift-rdu2-redhat-com:6443] in the remote context is configured with insecure-skip-tls-verify. If you agree to use that, pass in '--allow-skip-tls-verify true'. Aborting.
```

So I had to re-run the script and pass in the extra arg `--allow-skip-tls-verify true` 

Not sure if you want to mention this or not here in the docs. It was pretty self-explanatory when it happened, so at least the script's error message was helpful.

Reply:
Eh I probably won't mention it because I expect people's real clusters to have trusted certs already and like you mentioned the error is pretty self explanatory if they don't.